README.md

```diff
@@ -379,7 +379,7 @@ for integer in [4, 8, 15, 16, 23, 42] {
 }
 ```
 
-* **3.1.3** Prefer not declaring types for properties or constants if they can be inferred anyway.
+* **3.1.3** Prefer not declaring types for properties, constants, or variables if they can be inferred anyway.
```

Comment:
As above, do you think we need to list properties here if we already mentioned both variables and constants? I think that the original distinction here was a somewhat misworded one between instance variables and "static constants", but variables + constants cover those.